lib/nexpose/site.rb

```diff
@@ -516,9 +522,9 @@ def self.load(nsc, id)
       site.alerts = Alert.load_alerts(hash[:alerts])
       site.tags = Tag.load_tags(hash[:tags])
       site.web_credentials = hash[:web_credentials].map {|webCred| (
-                           webCred[:service] == Nexpose::WebCredentials::WebAppAuthType::HTTP_HEADER ?
-                               Nexpose::WebCredentials::Headers.new(webCred[:name], webCred[:baseURL], webCred[:soft403Pattern], webCred[:id]).object_from_hash(nsc,webCred) :
-                               Nexpose::WebCredentials::HTMLForms.new(webCred[:name], webCred[:baseURL], webCred[:loginURL], webCred[:soft403Pattern], webCred[:id]).object_from_hash(nsc,webCred))}
+      webCred[:service] == Nexpose::WebCredentials::WebAppAuthType::HTTP_HEADER ?
```

Comment:
I am not a fan of this whole block here. Needs some better formatting for readability. I'll merge for now, but I want to take another look at this before we release 1.0.